src/client/room.js

```diff
@@ -311,9 +311,18 @@ RoomMorph.prototype.createNewRole = function () {
                 'the provided name already exists.',
                 world
             );
+        } else if (roleName.includes('.')) {
```

Comment:
can you change this to use `indexOf` instead of `includes`? `includes` has limited support across browsers: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes